<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>561</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>v.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:40pt &quot;Lucida Calligraphy&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#004667;&quot;&gt;Agence de location des voitures&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGestion_de_parc_automobile">
    <property name="title">
     <string>Gestion de parc automobile</string>
    </property>
    <widget class="QMenu" name="menuMise_jour">
     <property name="title">
      <string>Mise à jour</string>
     </property>
     <widget class="QMenu" name="menuSupprimer_voiture">
      <property name="title">
       <string>Supprimer voiture</string>
      </property>
      <addaction name="actionSuppression_d_une_voiture_donn_e"/>
      <addaction name="actionSuppression_une_voiture_d_une_marque_donn_e"/>
      <addaction name="actionSuppression_des_voitres_age_10ans"/>
     </widget>
     <widget class="QMenu" name="menuModifier_voiture">
      <property name="title">
       <string>Modifier voiture</string>
      </property>
      <addaction name="actionPrix"/>
      <addaction name="actionCouleur"/>
      <addaction name="actionetat"/>
     </widget>
     <addaction name="actionAjouter_voiture"/>
     <addaction name="menuSupprimer_voiture"/>
     <addaction name="menuModifier_voiture"/>
    </widget>
    <widget class="QMenu" name="menuRecherche_affichage">
     <property name="title">
      <string>Recherche,affichage</string>
     </property>
     <addaction name="actionContenu_du_dictionaire_voitures"/>
     <addaction name="actionRecherche_par_matricule"/>
     <addaction name="actionRecherche_par_marque"/>
     <addaction name="actionRecherche_par_couleur"/>
     <addaction name="actionRecherche_des_voitures_disponibles"/>
     <addaction name="actionRecherche_des_voitures_lou_es"/>
    </widget>
    <addaction name="menuMise_jour"/>
    <addaction name="menuRecherche_affichage"/>
   </widget>
   <widget class="QMenu" name="menuGestion_des_clients">
    <property name="title">
     <string>Gestion des clients</string>
    </property>
    <widget class="QMenu" name="menuMise_jour_2">
     <property name="title">
      <string>Mise à jour</string>
     </property>
     <widget class="QMenu" name="menuModifier_les_donn_es_d_un_client">
      <property name="title">
       <string>Modifier les données d'un client</string>
      </property>
      <addaction name="actionadresse"/>
      <addaction name="actiontelephone"/>
      <addaction name="actionmail"/>
     </widget>
     <addaction name="actionAjouter_un_nouveau_client"/>
     <addaction name="actionSupprimer_un_client"/>
     <addaction name="menuModifier_les_donn_es_d_un_client"/>
    </widget>
    <widget class="QMenu" name="menuRecherche_affichage_2">
     <property name="title">
      <string>Recherche,affichage</string>
     </property>
     <addaction name="actioncontenu_du_dictionnaire_clients"/>
     <addaction name="actionRecherhe_par_CIN"/>
    </widget>
    <addaction name="menuMise_jour_2"/>
    <addaction name="menuRecherche_affichage_2"/>
   </widget>
   <widget class="QMenu" name="menuGestion_des_locations">
    <property name="title">
     <string>Gestion des locations</string>
    </property>
    <widget class="QMenu" name="menuMise_jour_3">
     <property name="title">
      <string>Mise à jour</string>
     </property>
     <widget class="QMenu" name="menuModifier_les_donn_es_d_une_location">
      <property name="title">
       <string>Modifier les données d'une location</string>
      </property>
      <addaction name="actionDate_de_location"/>
      <addaction name="actionDur_e"/>
     </widget>
     <addaction name="actionAjouter_une_nouvelle_locaztion"/>
     <addaction name="actionSupprimer_une_location"/>
     <addaction name="menuModifier_les_donn_es_d_une_location"/>
    </widget>
    <widget class="QMenu" name="menuRecherhce_affichage">
     <property name="title">
      <string>Recherhce, affichage</string>
     </property>
     <addaction name="actionContenu_du_dictionnaire_locations"/>
     <addaction name="actionRecherche_par_CIN"/>
     <addaction name="actionRecherche_par_matricule_2"/>
     <addaction name="actionRecherche_par_date_donn_e"/>
     <addaction name="actionRecherche_par_dur_e_de_location"/>
    </widget>
    <addaction name="menuMise_jour_3"/>
    <addaction name="menuRecherhce_affichage"/>
   </widget>
   <widget class="QMenu" name="menuEnregistrement_et_recuperation">
    <property name="title">
     <string>Enregistrement </string>
    </property>
    <addaction name="actionenregistrement_fichier_VOITURES"/>
    <addaction name="actionEnregistrement_fichier_CLIENTS"/>
    <addaction name="actionEnregistrement_fichier_LOCATIONS"/>
   </widget>
   <widget class="QMenu" name="menuAcuueil">
    <property name="title">
     <string>Acueil</string>
    </property>
   </widget>
   <addaction name="menuGestion_de_parc_automobile"/>
   <addaction name="menuGestion_des_clients"/>
   <addaction name="menuGestion_des_locations"/>
   <addaction name="menuEnregistrement_et_recuperation"/>
   <addaction name="menuAcuueil"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAjouter_voiture">
   <property name="text">
    <string>Ajouter voiture</string>
   </property>
  </action>
  <action name="actionSuppression_d_une_voiture_donn_e">
   <property name="text">
    <string>Suppression d'une voiture donnée</string>
   </property>
  </action>
  <action name="actionSuppression_une_voiture_d_une_marque_donn_e">
   <property name="text">
    <string>Suppression une voiture d'une marque donnée</string>
   </property>
  </action>
  <action name="actionSuppression_des_voitres_age_10ans">
   <property name="text">
    <string>Suppression des voitres(age&gt;10ans)</string>
   </property>
  </action>
  <action name="actionPrix">
   <property name="text">
    <string>Prix</string>
   </property>
  </action>
  <action name="actionCouleur">
   <property name="text">
    <string>Couleur</string>
   </property>
  </action>
  <action name="actionContenu_du_dictionaire_voitures">
   <property name="text">
    <string>Contenu du dictionaire voitures</string>
   </property>
  </action>
  <action name="actionRecherche_par_matricule">
   <property name="text">
    <string>Recherche par matricule</string>
   </property>
  </action>
  <action name="actionRecherche_par_marque">
   <property name="text">
    <string>Recherche par marque</string>
   </property>
  </action>
  <action name="actionRecherche_par_couleur">
   <property name="text">
    <string>Recherche par couleur</string>
   </property>
  </action>
  <action name="actionRecherche_des_voitures_disponibles">
   <property name="text">
    <string>Recherche des voitures disponibles</string>
   </property>
  </action>
  <action name="actionRecherche_des_voitures_lou_es">
   <property name="text">
    <string>Recherche des voitures louées</string>
   </property>
  </action>
  <action name="actionAjouter_un_nouveau_client">
   <property name="text">
    <string>Ajouter un nouveau client</string>
   </property>
  </action>
  <action name="actionSupprimer_un_client">
   <property name="text">
    <string>Supprimer un client</string>
   </property>
  </action>
  <action name="actionadresse">
   <property name="text">
    <string>adresse</string>
   </property>
  </action>
  <action name="actiontelephone">
   <property name="text">
    <string>telephone</string>
   </property>
  </action>
  <action name="actionmail">
   <property name="text">
    <string>mail</string>
   </property>
  </action>
  <action name="actioncontenu_du_dictionnaire_clients">
   <property name="text">
    <string>contenu du dictionnaire clients</string>
   </property>
  </action>
  <action name="actionRecherhe_par_CIN">
   <property name="text">
    <string>Recherche par CIN</string>
   </property>
  </action>
  <action name="actionAjouter_une_nouvelle_locaztion">
   <property name="text">
    <string>Ajouter une nouvelle location</string>
   </property>
  </action>
  <action name="actionSupprimer_une_location">
   <property name="text">
    <string>Supprimer une location</string>
   </property>
  </action>
  <action name="actionDate_de_location">
   <property name="text">
    <string>Date de location</string>
   </property>
  </action>
  <action name="actionDur_e">
   <property name="text">
    <string>Durée</string>
   </property>
  </action>
  <action name="actionContenu_du_dictionnaire_locations">
   <property name="text">
    <string>Contenu du dictionnaire locations</string>
   </property>
  </action>
  <action name="actionRecherche_par_CIN">
   <property name="text">
    <string>Recherche par CIN</string>
   </property>
  </action>
  <action name="actionRecherche_par_matricule_2">
   <property name="text">
    <string>Recherche par matricule</string>
   </property>
  </action>
  <action name="actionRecherche_par_date_donn_e">
   <property name="text">
    <string>Recherche par date donnée</string>
   </property>
  </action>
  <action name="actionRecherche_par_dur_e_de_location">
   <property name="text">
    <string>Recherche par durée de location</string>
   </property>
  </action>
  <action name="actionRecherche_des_locations_entre_2_dates_donn_es">
   <property name="text">
    <string>Recherche des locations entre 2 dates données</string>
   </property>
  </action>
  <action name="actionenregistrement_fichier_VOITURES">
   <property name="text">
    <string>enregistrement fichier VOITURES</string>
   </property>
  </action>
  <action name="actionRecuperation_fichier_VOITURES">
   <property name="text">
    <string>Recuperation fichier VOITURES</string>
   </property>
  </action>
  <action name="actionEnregistrement_fichier_CLIENTS">
   <property name="text">
    <string>Enregistrement fichier CLIENTS</string>
   </property>
  </action>
  <action name="actionRecuperation_fichier_CLIENTS">
   <property name="text">
    <string>Recuperation fichier CLIENTS</string>
   </property>
  </action>
  <action name="actionEnregistrement_fichier_LOCATIONS">
   <property name="text">
    <string>Enregistrement fichier LOCATIONS</string>
   </property>
  </action>
  <action name="actionRecuperation_fichier_LOCATIONS">
   <property name="text">
    <string>Recuperation fichier LOCATIONS</string>
   </property>
  </action>
  <action name="actionetat">
   <property name="text">
    <string>etat</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res1.qrc"/>
 </resources>
 <connections/>
</ui>
